com.github.onegal.Viper.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: com.github.onegal.Viper
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: viper-run
+separate-locales: false
+tags: []
+finish-args:
+  - --socket=x11 # Allows us to create windows
+  - --share=ipc # Necessary for X11
+  - --share=network # Network access
+  - --device=dri # OpenGL rendering
+  - --filesystem=host # Full filesystem access, Titanfall2 could be installed anywhere on disk
```

Comment:
:-(

Reply:
What's the issue here exactly? As explained in the initial description`--filesystem=host` is sadly necessary :/